courses/migrations/0013_backfill_paid_courserun.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.14 on 2022-07-28 20:52
+
+from django.db import migrations
+
+
+def backfill_paidcourserun(apps, schema_editor):
+    """Backfill PaidCourseRun for existing orders"""
+    PaidCourseRun = apps.get_model("courses", "PaidCourseRun")
+    CourseRun = apps.get_model("courses", "CourseRun")
+    Order = apps.get_model("ecommerce", "Order")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    for order in Order.objects.filter(state__in=["fulfilled", "review"]):
```

Comment:
```suggestion
    for order in Order.objects.filter(state__in=[Order.STATE.FULFILLED, Order.STATE.REVIEW]):
```

Reply:
I tried it with Order.STATE.FULFILLED, It says "AttributeError: type object 'Order' has no attribute 'STATE'". By debugging it, it seems like these constant are not available in app defined model - Order = apps.get_model("ecommerce", "Order").
I could import Order model directly, then Order.STATE.FULFILLED is available, but from what I read, we should always use app model.  Please let me know if I am missing anything

Reply:
Yeah, We should always use `apps.get_model`.  You are right. If this is the case then I am okay with this one. You can ignore my comment. Thanks

Reply:
@jkachel What's your say on this? I think we can go without model constants.

Reply:
After some more thought, we actually shouldn't be using the constants here. Future revisions may change the values of the constants and, if that happens, those changes would invalidate this migration. So, these should just be hard-coded in order to make the migration idempotent.